### PR TITLE
Add Golang Buildpack Health Check

### PR DESCRIPTION
**Pull request by micahlee**
Connected to #101

This PR adds a health check using Go and the Conjur Go API to verify the connection settings will work with the Conjur buildpack.

Additionally, 6a2ce31 initializes the rubocop configuration and TODO list for the service broker. These issues will be address in #109.

### CODE REVIEW

File: buildpack-health-check/main.go [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+	"os"
+
+	"github.com/cyberark/conjur-api-go/conjurapi"
+	"github.com/cyberark/conjur-api-go/conjurapi/authn"
+)
+
+func main() {
+	followerURL := os.Getenv("CONJUR_FOLLOWER_URL")
+	if len(followerURL) > 0 {
+		os.Setenv("CONJUR_APPLIANCE_URL", followerURL)
```
Comment (jtuttle) [inline buildpack-health-check/main.go]: This is getting set in the service broker environment I believe? Is it okay to leave it around or should we clean it up?

File: buildpack-health-check/main.go [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+	printAndExitIfError(err)
+
+	fmt.Println("Successfully connected to Conjur.")
+}
+
+func printAndExitIfError(err error) {
+	if err == nil {
+		return
+	}
+	os.Stderr.Write([]byte("There is an issue with your Conjur configuration.\n"))
```
Comment (jtuttle) [inline buildpack-health-check/main.go]: There appears to be a [WriteString method](https://golang.org/pkg/os/#File.WriteString) that might be cleaner than dealing w/ byte arrays.

File: buildpack-health-check/main.go (hunk 1 above)
Comment (micahlee) [reply buildpack-health-check/main.go]: You are referring to the `CONJUR_APPLIANCE_URL` environment variable, correct?

No, this only sets it in the context of the `buildpack-health-check` execution, and doesn't change the environment of the calling script, `start-service-broker.sh`. The service broker will still see the `CONJUR_APPLIANCE_URL` as configured in the tile.

File: buildpack-health-check/main.go (hunk 2 above)
Comment (micahlee) [reply buildpack-health-check/main.go]: Cool, thanks! I will make the change.

**Pull request merged**